build/building/context.md

```diff
@@ -150,7 +158,36 @@ RUN --mount=target=. \
 ```
 
 ```console
-$ docker build --build-arg BUILDKIT_CONTEXT_KEEP_GIT_DIR=1 https://github.com/user/myrepo.git#main
+$ docker build \
+  --build-arg BUILDKIT_CONTEXT_KEEP_GIT_DIR=1
+  https://github.com/user/myrepo.git#main
+```
+
+#### Private repositories
+
+When you use a Git repository as the build context, the Buildx client
+downloads the repository to a temporary directory on its host before
+sending it to the BuildKit daemon.
```

Comment:
I don't think this is quite right :thinking: BuildKit is actually the one doing the clone here.

Reply:
Hm... the original text actually seems wrong :eyes:

@crazy-max I'm not misremembering something here right? BuildKit *definitely* does the clone right?

Reply:
I removed these bits, they're not important anyway